src/Hypergrid/modules.ts

```diff
@@ -13,7 +13,7 @@
 // overridable modules
 // Hypergrid vectors through here for these modules
 module.exports = {
-    Scrollbar: require('finbars'),
+    Scrollbar: require('../finbars/finbars'),
```

Comment:
Removed the `finbars` dependency, and instead copied and modified the file in `src/finbars/finbars.ts`.